requirements.txt

```diff
@@ -2,3 +2,4 @@ bokeh==3.4.1
 pandas==2.2.1
 tbparse==0.0.8
 toml==0.10.2
+kubernetes==30.1.0
```

Comment:
We also need to add it into `pyproject.toml`, otherwise it be automatically installed during building the package.

Reply:
Resolved by @amaslenn's PR: https://github.com/NVIDIA/cloudai/pull/190